process/block/sovereignChainBlock.go

```diff
@@ -37,23 +39,31 @@ type sovereignChainBlockProcessor struct {
 	extendedShardHeaderTracker   extendedShardHeaderTrackHandler
 	extendedShardHeaderRequester extendedShardHeaderRequestHandler
 	chRcvAllExtendedShardHdrs    chan bool
+	outgoingOperationsFormatter  sovereign.OutgoingOperationsFormatter
+}
+
+type ArgsSovereignChainBlockProcessor struct {
```

Comment:
Comment for this is already added in  https://github.com/multiversx/mx-chain-go/pull/5673